### ISSUE TITLE
Project still alive? Test page uses extremely old data files

### ISSUE THREAD

**Issue by wowcut** [ThaDafinser/UserAgentParser#118]
Is this project still alive? On the test page I see tat the underlying data was not update since over two years... what looks like a "project dead" indicator to me...

**Comment by ThaDafinser** [comment]
Yeah, it definitely needs an update. In general the package should still work fine.
